dev/com.ibm.websphere.org.eclipse.microprofile.config.1.2/bnd.bnd

```diff
@@ -21,8 +21,8 @@ Import-Package: \
   *
 
 Export-Package: \
-  org.eclipse.microprofile.config;version=1.0, \
-  org.eclipse.microprofile.config.inject;version=1.0, \
+  org.eclipse.microprofile.config;version=1.1, \
```

Comment:
I thought we agreed the package version will stay as 1.0, which is correct one.

Reply:
you're still looking at old code